.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```

Comment:
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/38065883/173183552-38fcdca4-fbe2-4877-b148-d97ddc09d359.png">
It's better to use `flex-start` instead of `space-between`, so the structure will remain even if there are not enough cards